helm/csm-replication/values.yaml

```diff
@@ -7,6 +7,10 @@ replicas: 1
 # Allowed values: string
 image: dellemc/dell-replication-controller:v1.3.1
 
+# image: Defines controller's init container image. This shouldn't be changed
+# Allowed values: string
+initImage: dellemc/dell-replication-init:v1.0
```

Comment:
Set tag to v1.0.0 to follow semantic versioning

Reply:
Updated. Thanks!